samples/Mvc.Formatters/Mvc.Formatters/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@{
+    ViewData["Title"] = "Posting data in text/plain format";
+}
+<h2>Posting data in text/plain format</h2>
+<div>
+    <label>Sample text:</label>
+    <input type="text" id="sampleText" />
+    <button id="submitText" onclick="javascript: submitData()">Submit</button>
```

Comment:
nit: onclick is from the nineties. It's considered a bad practice these days to do that. You can do 
```
$(function(){
  $('#submitText').on('click', submitData);
});
```

If you keep the JavaScript approach, at least change it to be from this century :smile: